Extension/package.json

```diff
@@ -1563,7 +1563,7 @@
     "ts-loader": "^6.0.4",
     "tslint": "^5.19.0",
     "typescript": "^3.5.3",
-    "vscode-cpptools": "^3.1.0",
+    "vscode-cpptools": "^4.0.1",
```

Comment:
I think you will need to update the version in package.json to 0.28.3-master

Reply:
I created a separate PR for updating the version in release for 0.28.3 patch.